From c41ea08127d8cda6b95c828089b8f1df74335a14 Mon Sep 17 00:00:00 2001
From: Jan Vermaete <jan.vermaete@gmail.com>
Date: Wed, 24 Apr 2024 20:32:45 +0200
Subject: [PATCH 10/10] WIP: test: filter tests that should not run on the
 target

Do not the tests marked with !CROSS_COMPILED

These tests are skipped when running the OpenDDS testsuite on a target.

Upstream-Status: Pending

Signed-off-by: Jan Vermaete <jan.vermaete@gmail.com>
---
 tests/dcps_tests.lst | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests/dcps_tests.lst b/tests/dcps_tests.lst
index efe998a25..7a0040014 100644
--- a/tests/dcps_tests.lst
+++ b/tests/dcps_tests.lst
@@ -77,12 +77,12 @@ tests/DCPS/Compiler/is_topic_type/run_test.pl: !DCPS_MIN
 tests/DCPS/Compiler/vread_vwrite/run_test.pl: !DCPS_MIN RAPIDJSON CXX11
 tests/DCPS/Compiler/TryConstruct/run_test.pl: !DCPS_MIN
 tests/DCPS/Compiler/TryConstruct/C++11/run_test.pl: !DCPS_MIN CXX11 !OPENDDS_SAFETY_PROFILE
-tests/DCPS/Compiler/typeobject_hash_consistency/run_test.pl: !DCPS_MIN !OPENDDS_SAFETY_PROFILE
+tests/DCPS/Compiler/typeobject_hash_consistency/run_test.pl: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !CROSS_COMPILED
 tests/DCPS/Compiler/xcdr/run_test.pl
 tests/DCPS/Compiler/xcdr/run_test.pl --dynamic: !OPENDDS_SAFETY_PROFILE !DDS_NO_CONTENT_SUBSCRIPTION
 tests/DCPS/Compiler/XtypesExtensibility/run_test.pl: !DCPS_MIN
 tests/DCPS/Compiler/typeobject_generator/run_test.pl
-tests/DCPS/Compiler/keywords/run_test.pl classic: !DCPS_MIN
+tests/DCPS/Compiler/keywords/run_test.pl classic: !DCPS_MIN !CROSS_COMPILED
 tests/DCPS/Compiler/keywords/run_test.pl cpp11: !DCPS_MIN CXX11 !OPENDDS_SAFETY_PROFILE
 tests/DCPS/Compiler/explicit_ints/run_test.pl classic: !DCPS_MIN
 tests/DCPS/Compiler/explicit_ints/run_test.pl cpp11: !DCPS_MIN CXX11 !OPENDDS_SAFETY_PROFILE
@@ -266,7 +266,7 @@ tests/stress-tests/run_test.pl: !DCPS_MIN !NO_UNIT_TESTS
 tests/DCPS/KeyTest/run_test.pl keymarshalling: !DCPS_MIN
 tests/DCPS/KeyTest/run_test.pl isbounded: !DCPS_MIN
 tests/DCPS/KeyTest/run_test.pl md5: !DCPS_MIN
-tests/DCPS/KeyTest/run_test.pl compiler: !DCPS_MIN !TARGET !OPENDDS_SAFETY_PROFILE
+tests/DCPS/KeyTest/run_test.pl compiler: !DCPS_MIN !TARGET !OPENDDS_SAFETY_PROFILE  !CROSS_COMPILED
 tests/DCPS/CompatibilityTest/run_test.pl: !DCPS_MIN !OPENDDS_SAFETY_PROFILE
 tests/DCPS/CompatibilityTest/run_test.pl rtps_disc: !DCPS_MIN RTPS !GH_ACTIONS_M10
 tests/DCPS/CompatibilityTest/run_test.pl rtps_disc_tcp: !DCPS_MIN RTPS !OPENDDS_SAFETY_PROFILE
-- 
2.39.2

